@here/harp-datasource-protocol/lib/Theme.ts

```diff
@@ -196,17 +191,34 @@ export interface StylePriority {
 }
 
 /**
- * A type representing HARP themes with all the styleset declarations
- * grouped in one `Array`.
- *
- * @internal This type will merge with {@link Theme}.
+ * Map theme is used to define what features are shown and how the map is styled, for example
+ * which lightning is used or whether fog should be displayed.
  */
-export type FlatTheme = Omit<Theme, "styles"> & {
+export interface Theme extends BaseTheme {
```

Comment:
this should probably be renamed to LegacyTheme and marked as deprecate.

Reply:
I agree, I think we should update Theme to be the FlatTheme and deprecate the former as something like LegacyTheme... have you seen: HARP-12835